fedora/client/openidcclient.py

```diff
@@ -331,32 +311,68 @@ def token_app(environ, start_response):
     def get_token(self, scopes, new_token=True):
         """Function to retrieve tokens with specific scopes.
 
-        This function will block until a token is retrieved.
+        This function will block until a token is retrieved if requested.
```

Comment:
In that case I think the documentation needs to be re-worked to make that clear. Rather than "if requested", maybe something like "if ``new_token`` is true and there is no existing token". It might be better to call that keyword ``block`` or something, though.